memcached-operator/deploy/olm-catalog/memcached-operator/0.12.0/memcached-operator.v0.12.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,177 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    olm.skipRange: '>=0.11.0'
```

Comment:
@estroz

Reply:
If you remove `spec.replaces` then `olm.skipRange` is unnecessary.

Reply:
make sense .. backing the replace one.